### PR TITLE
chore: share panel color fix

### PR DESCRIPTION
**Pull request by milljoniaer**
### This should fix the colors of the share and follow bar at the end of the blog posts:
- dark blue for the labels
- correct primary blue for the icons

### Browse the preview
https://satellytescommain21751-fixcolorblog.gtsb.io/blog/boosting-developer-productivity-with-github-actions/

image

#395

### CODE REVIEW

File: src/components/legacy/icons/social-icons/whatsapp.tsx [hunk 1]
```diff
@@ -9,15 +9,15 @@ export const IconWhatsapp: React.FC<WhatsappProps> = (props) => (
...
       d="M12.3179 23.1044C18.7694 23.1044 23.9993 17.9323 23.9993 11.5522C23.9993 5.17209 18.7694 0 12.3179 0C5.86644 0 0.63648 5.17209 0.63648 11.5522C0.63648 13.5581 1.15348 15.4447 2.06289 17.0886L0 23.0766L5.73911 21.0995C7.61277 22.3646 9.87798 23.1044 12.3179 23.1044ZM17.7617 15.5657C17.7736 15.6569 17.7591 15.7492 17.7197 15.8316C17.5878 16.1072 17.0232 16.6262 16.0693 16.8465C14.9872 17.0965 12.7296 17.0732 9.76508 14.1087C6.80042 11.144 6.7771 8.88667 7.02697 7.80506C7.2472 6.8507 7.76626 6.28568 8.04193 6.15385C8.12419 6.11455 8.21649 6.10002 8.30764 6.11192C8.57717 6.14691 9.17502 6.22881 9.48954 6.27211L9.59379 6.28641C9.74869 6.3077 9.88447 6.40331 9.95757 6.54174C10.1227 6.85545 10.6799 7.97174 10.9168 9.16368C10.9422 9.2919 10.9142 9.42429 10.8391 9.52988L10.0831 10.5942C10.2174 10.9074 10.613 11.7196 11.3836 12.4902C12.1622 13.2688 12.9658 13.6582 13.2787 13.791L14.3436 13.0344C14.4491 12.9594 14.5816 12.9313 14.71 12.9568C15.902 13.1938 17.0182 13.751 17.3319 13.9161C17.4705 13.9896 17.5659 14.1252 17.5872 14.2798L17.5919 14.3137C17.6311 14.5995 17.7236 15.2746 17.7617 15.5657Z"
-      fill={props.color ? props.color : '#668CFF'}
+      fill="currentColor"
```
Comment (georgiee) [inline src/components/legacy/icons/social-icons/whatsapp.tsx:11]: that's the right approach. Ideally we would get rid of those legacy icons and use the icon component instead?

**Review by georgiee — CHANGES_REQUESTED** [review]
Thanks looks good! I would love to see the icons migrated to the icon component format with this PR. Could you do that? Otherwise the changes look good!

File: src/components/legacy/icons/social-icons/whatsapp.tsx (hunk 1 above)
Comment (milljoniaer) [reply src/components/legacy/icons/social-icons/whatsapp.tsx:11]: thank you for your suggestion. Please have another look. I used the Icon Component in both, the share panels and the social links in the navigation

**Review by feedm3 — APPROVED** [review]
Very good, all icons are now in their separate files. Let's merge this 🚀

File: src/assets/icons/twitter.svg [hunk 2]
```diff
@@ -1,31 +1,13 @@
...
 
-interface TwitterProps {
-  color?: string;
-}
-
-export const IconTwitter: React.FC<TwitterProps> = (props) => (
-  <svg
-    width="27"
-    height="22"
     viewBox="0 0 27 22"
```
Comment (georgiee) [inline src/assets/icons/twitter.svg]: a bounding box of `0 0 24 24` is required to properly align. You can't just change the numbers as the underlying vectors don't fit the 1:1 aspect ratio.  I think it's best to ask Verena for help. Usually you create a square of 24x24px in your favourite illustration app and insert and scale the icons so it's properly aligned.

This issue happens in the other icons too.

Screenshot 2022-02-25 at 09 03 16

**Review by georgiee — CHANGES_REQUESTED** [review]
thanks for moving the icons 💫
Please fix the alignment though by talking to Verena so you could the right assets 🙏

File: src/assets/icons/email-24.svg [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (georgiee) [inline src/assets/icons/email-24.svg:1]: looks good 👍

File: src/assets/icons/linked-in-24.svg [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (georgiee) [inline src/assets/icons/linked-in-24.svg:1]: 👍

File: src/assets/icons/twitter-24.svg [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (georgiee) [inline src/assets/icons/twitter-24.svg:1]: 👍

**Review by georgiee — APPROVED** [review]
icons are updated and it looks properly aligned. thank you, let's merge!

Screenshot 2022-02-25 at 10 54 25

**Pull request merged**